tests/acm_observability/test_acm_observability.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from const import HUB_CLUSTER
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+pytestmark = pytest.mark.acm_observability
+
+
+class TestACMObservability:
+    def test_hub_etcd_metrics_exists(
+        self,
+        admin_client_scope_session,
+        clusters_etcd_metrics,
+    ):
+        assert isinstance(
+            clusters_etcd_metrics[HUB_CLUSTER][-1], float
```

Comment:
is `0.00` valid or do we expect something greater than that?

Reply:
good call, we expect a greater value. will update